### DISCUSSION TITLE
Understanding healthcare cost per patient

### DISCUSSION THREAD

**Discussion by sestes6** [synthetichealth/synthea#1469] [Q&A] [answered]
Hello,
My question involves the Healthcare_Expenses and Healthcare_Coverage fields in the patients.csv file [here](https://github.com/synthetichealth/synthea/wiki/CSV-File-Data-Dictionary#patients). I thought that I could sum up the healthcare costs of a patient in some of the other tables and reproduce this number. In particular, it seems that just about every medical/healthcare-related event for a patient is listed in the encounters.csv file. So I thought that I could just sum up the total_claim_cost from every encounter a patient had and that should equal the sum of those two quantities (healthcare expenses and coverage fields). The sum of the total claim cost seems to be lower for every patient (in some cases by a factor of 10 or more). This is confusing to me. I would have thought that the sum of these numbers represents the cost of all medical events throught the patient's life. Is this not correct? Or are the other tables not comprehensive and there are some "hidden costs" throughout the patient's lifetime that don't appear in the encounters.csv table? I hope this question makes sense. I would really appreciate any help that the community can provide. Thanks!

**Comment by marketingteamhealthtech** [comment]
Bluebrix provides technology solutions that promote value-based care, watered with distinct business and care models. Through its sophisticated technology backbone, the company ensures that all the data analysis is taken care of while providing for patient engagement methods and other operational processes that assist in the realization of good health results and efficient operations. Personalised, affordable healthcare is possible with our technology platform in place so that medical doctors can treat well at lower costs without wasting resources on over-prescription yet more than 90% feel satisfied by our e-services.
https://bluebrix.health/healthcare-delivery/

**Comment by jawalonoski** [comment, marked as answer]
@sestes6 the summation may not match depend on the years of history you've exported (default is 10 years, with some exceptions: for example, **active** diagnoses, devices, prescriptions, and so forth that are older than the filter may be included).

If some of the data was filtered as too old, the total in the patient file would be higher than the remaining individual items that you would add together.

See `src/main/resources/synthea.properties`:

```properties
# number of years of history to keep in exported records, anything older than this may be filtered out
# set years_of_history = 0 to skip filtering altogether and keep the entire history
exporter.years_of_history = 10
```

**Reply by sestes6** [reply]
Ah thank you so much, that is very helpful! In particular, the data I'm working with comes from the 100 patient sample [here](https://synthea.mitre.org/downloads). I'm guessing that this dataset was generated with the default value of `10` that you mentioned. Is that correct?

**Reply by jawalonoski** [reply]
Yes, the samples are generated with default values.

**Reply by sestes6** [reply]
Thank you!
